### PR TITLE
Fixes #374

### LINKED ISSUE #374: CVScores
Implement a visualizer that shows cross-validation scores as a bar chart along with the final score as an annotated horizontal line. 

We want to start moving toward better cross-validation and model selection. Create `yb.model_selection.CVScores` visualizer that extends `ModelVisualizer` and wraps an estimator. Accepts a `cv` and `scoring` params, similar to the ones exposed in [`sklearn.model_selection.cross_val_score`](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_val_score.html). 

Output is a bar chart with the score of each split and a horizontal dotted annotation with the average score.

### PR DESCRIPTION
**Pull request by pdamodaran**
Fixes #

[Description]

Changes proposed in this pull request:

- added documentation
- removed duplicate line
-

@DistrictDataLabs/team-oz-maintainers

### CODE REVIEW

File: yellowbrick/model_selection/CVScores.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        self.ax.legend(frameon=True, loc=loc, edgecolor=edgecolor)
+
+        #set spacing between the x ticks
+        self.ax.xaxis.set_major_locator(ticker.MultipleLocator(1))
+
+        # Set the axis labels
+        self.ax.set_xlabel('Training Instances')
+        self.ax.set_ylabel('Score')
+
+
```
Comment (lwgray) [inline yellowbrick/model_selection/CVScores.py]: nice work on the visualizer.  I caught a couple of things.  Let me know if you need me to clarify any of this.

1.  CVScores.py is missing a "quick method".  Look at the method found at end of the validationcurve.py code for an example of what I mean

Comment (lwgray) [reply yellowbrick/model_selection/CVScores.py]: 2.  The CVScore method isn't callable.  Meaning I tried to run the example with your code and I get an error.  The solution is to edit the __init__.py file that is in the same directory as CVScores.py.  You need to add an import statement similar to the ones already present in __init__.py

**Comment by lwgray** [comment]
I left you some code comments, let me know if I need to better explain.

**Comment by bbengfort** [comment]
@pdamodaran @lwgray I'm going to close this PR in favor of #444 -- we don't want to merge to master; but instead to develop. @pdamodaran  you can just push commits to your fork's master and they should show up in #444; though we may need to deal with the rebase issue of merging from develop. 

@lwgray would you mind copying your comments over to #444?

**Pull request closed without merging**